PowerEditor/installer/nsisInclude/GetVersion.nsi

```diff
@@ -0,0 +1,70 @@
+!include "FileFunc.nsh"
+!include "logiclib.nsh"
+
+OutFile "GetVersion.exe"
+SetCompressor /SOLID lzma 		; Compression technique
+RequestExecutionLevel user 		; This is to make output exe run as invoker (Admin privileges not required)
+!define EXEPATH "..\bin\Notepad++.exe"	; Absolute path
+
+
+Section "Main"
+	SetAutoClose true	; it simply writes version info in the text. So close immediatly after writing version info
+
+    DetailPrint "Getting version from ${EXEPATH}"
+    GetDllVersion "${EXEPATH}" $R0 $R1 	; Two values were read during compilation
+	
+	;${GetFileVersion} "${EXEPATH}" $R0
+	;DetailPrint "Another way to get version: $R0"
+
+    IntOp $R2 $R0 / 0x00010000		; $R2 now contains major version
+    IntOp $R3 $R0 & 0x0000FFFF		; $R3 now contains minor version
+    IntOp $R4 $R1 / 0x00010000		; $R4 now contains release
+    IntOp $R5 $R1 & 0x0000FFFF		; $R5 now contains build
+	
+	; if notepad++.exe version = 7.0.0.0
+	; 	$1 = 7.0.0.0 (File Version)
+	;	$2 = 7		 (Product Version)
+	;	$3 = 0		 (Minor Version)
+	
+	; if notepad++.exe version = 7.1.2.0
+	; 	$1 = 7.0.0.0 (File Version)
+	;	$2 = 7.1.2 	 (Product Version)
+	;	$3 = 12		 (Minor Version)
+	
```

Comment:
What Windows version is that?
Strange that it shows the digital/fixed value for File version and textual/variable for Product version.

Yep, I've used automatic generated resource info in projects based on the linked MS info.
Notepad++ version numbering is not the same as MS tends to do it.
Here are some different methods I've seen over the years:
```
- Windows uses       : major.minor[.build[.revision]]
- .NET    uses       : major.minor[.revision[.build]]
- other used formats : major.minor[.maintenance[.build]]
                       major.minor[.build[.((majorRevision && 0x00FF)<<8)||(minorRevision && 0x00FF)]]
                       major.minor[.patch[.stage]]
```
More recent I've encountered projects mentioning they use [Semantic Versioning](http://semver.org/).